README.md

```diff
@@ -586,13 +586,16 @@ The IP configuration supports the following options:
 
   Static route configuration can be specified via a list of routes given in the
   `route` option. The default value is an empty list. Each route is a dictionary with
-  the following entries: `network`, `prefix`, `gateway`, `metric` and `table`.
+  the following entries: `gateway`, `metric`, `network`, `prefix`, `table` and `type`.
   `network` and `prefix` specify the destination network. `table` supports both the
-  numeric table and named table. In order to specify the named table, the users have
-  to ensure the named table is properly defined in `/etc/iproute2/rt_tables` or
-  `/etc/iproute2/rt_tables.d/*.conf`.
-  Note that Classless inter-domain routing (CIDR) notation or network mask notation
-  are not supported yet.
+  numeric table and named table. In order to specify the named table, the users have to
+  ensure the named table is properly defined in `/etc/iproute2/rt_tables` or
+  `/etc/iproute2/rt_tables.d/*.conf`. The supported `type` are `blackhole`, `prohibit`,
+  `unreachable` and these three type routes can not have next hop (gateway) specified.
+  It is possible to create a route without `type` specified. For the route `type`
+  definition, please refer to [man 8 ip-route](https://man7.org/linux/man-pages/man8/ip-route.8.html#DESCRIPTION).
+  Note that Classless inter-domain routing(CIDR) notation or network mask notation are
+  not supported yet.
```

Comment:
The text does not flow. Here is another proposal:

```suggestion
  `/etc/iproute2/rt_tables.d/*.conf`. The optional `type` key supports the values `blackhole`, `prohibit`,
  and `unreachable`, see [man 8 ip-route](https://man7.org/linux/man-pages/man8/ip-route.8.html#DESCRIPTION) for their definition. Routes with these types do not support a gateway. If the type is not specified, the route is considered a unicast route.
  
 Note that the classless inter-domain routing(CIDR) notation or the network mask notation are
  not supported for the `network` key.
```

Is the statement regarding unicast routes correct?
Please

Reply:
I think that the statement regarding the unicast route is correct.

Reply:
I do not see a reaction to my proposal, therefore I unresolved this conversation.

Reply:
Sorry, I forgot to push the changes last night.

Reply:
Your proposal looks quite nice to me, Thanks.